### PR TITLE
Use prop-types instead of proptypes, and files reorganization

### LINKED ISSUE #135: Cropped avatar on mobile in the top bar
A picture > a thousand words

**Screenshots**

### LINKED ISSUE #149: Change proptypes to prop-types
We should use the [`prop-types`](https://www.npmjs.com/package/prop-types) instead of the [`proptypes`](https://www.npmjs.com/package/proptypes) package, as it's the one mentionned in the [ReactJS documentation](https://reactjs.org/docs/typechecking-with-proptypes.html) and it's maintained.

### PR DESCRIPTION
**Pull request by nathanaelhoun**
## Changes

- [x] Closes #135 
- [x] Closes #149 
- [x] Updated files organization
- [x] Fix non-valid HTML markup in About page
- [x] Some CSS cleanup
- [x] Update Train test to fix errors raised by proptypes

This is way cleaner ! 
image

### CODE REVIEW

**Review by valentinperignon — APPROVED** [review]
LGTM!

File: client/src/pages/About.js [hunk 1]
```diff
@@ -68,23 +68,21 @@ export default class About extends Component {
...
-              <a href="https://icons.modulz.app/" target="_blank" rel="noreferrer">
-                Modulz
-              </a>
-              .
-            </li>
-          </ol>
-        </p>
+        <ol id="credits">
+          <li>Réalisé par Nathanaël Houn, François Poguet et Valentin Perignon.</li>
+          <li>Supervisé par Anne-Laure Claire, Frédéric Dadeau, et Marc Pudlo.</li>
```
Comment (FrancoisPog) [inline client/src/pages/About.js:73]: Anne-Laure Claire**t** 🤓

**Review by FrancoisPog — APPROVED** [review]
Just need to correct Anne-Laure Clairet's name, but LGTM!

File: client/src/pages/About.js (hunk 1 above)
Comment (nathanaelhoun) [reply client/src/pages/About.js:73]:
```suggestion
          <li>Supervisé par Anne-Laure Clairet, Frédéric Dadeau, et Marc Pudlo.</li>
```

**Pull request merged**